src/chat/channel.py

```diff
@@ -199,7 +199,12 @@ def pingWindow(self):
     def openUrl(self, url):
         logger.debug("Clicked on URL: " + url.toString())
         if url.scheme() == "faflive":
-            replay(url)
+            if replay(url) == "Replays":  # game ended  -> Search Replaya
+                # hack to get player name from url
+                player_id = int(url.path().split("/")[2].split(".")[0])
+                if player_id in self._chatterset._playerset._players:
+                    player = self._chatterset._playerset._players[player_id].login
```

Comment:
Holy shit, these two lines. Class members starting with an underscore are private, this should be respected. If we need a playerset in the class, pass it in the constructor. Playerset itself has dict-like semantics, there are plenty examples for it in existing code.

If we make a dedicated class for handling these URLs, then using playerset directly won't even be necessary.